src/widgets/survey/ui/Intermediate.tsx

```diff
@@ -133,6 +133,7 @@ function Intermediate({
         eventId,
         pipelineActivityOrder: activitiesPassed,
         totalActivities,
+        flowName: flowName || '[Name unknown]',
```

Comment:
We don't need to change flow name during switch to another activity. Remove the line. Adjust the payload types if needed.